src/view/background/BaseBackgroundPainter.cpp

```diff
@@ -23,9 +51,8 @@ void BaseBackgroundPainter::paint(cairo_t* cr, PageRef page, BackgroundConfig* c
     this->width = page->getWidth();
     this->height = page->getHeight();
 
-    uint32_t temp{};
-    this->foregroundColor1 = this->config->loadValueHex("f1", temp) ? temp : this->foregroundColor1;
-    this->foregroundColor2 = this->config->loadValueHex("f2", temp) ? temp : this->foregroundColor2;
+    this->foregroundColor1 = this->getForegroundColor1();
+    this->foregroundColor2 = this->getForegroundColor2();
```

Comment:
This is done because not all templates have to define foreground colors. If the template doesn't define the colors, the default background painter will be used which is set in ```resetConfig()``` of the respective background painter. I agree that it isn't very clear and that the default color vars could be renamed.